sklearn/tree/_classes.py

```diff
@@ -1472,6 +1472,21 @@ class ExtraTreeClassifier(DecisionTreeClassifier):
     reduce memory consumption, the complexity and size of the trees should be
     controlled by setting those parameter values.
 
+    Examples
```

Comment:
The example is good. However, the `Examples` section should come after the `References` section to follow the validation of `numpydoc`. Could you move the section a bit down?

Reply:
Thanks!